### PR TITLE
Fix issue of exporting to google map multiple polygon

### PR DESCRIPTION
**Pull request by songjiahong**
1. Fix the algorithm of checking whether a polygon is clockwise
2. Add all vertexes including the last overlapped one to the google map polygon

### CODE REVIEW

**Comment by arthur-e** [comment]
It looks like several tests are failing. Can you check out the Travis CI build details? In particular, the first Google Maps 3 test that fails:

```
FAILED TESTS:
  Standard WKT Test Cases: 
    Coverting WKT strings into objects: 
      ✖ should convert a basic POLYGON string to a Polygon instance
        PhantomJS 2.1.1 (Linux 0.0.0)
      Expected '(10, 30),(40, 40),(40, 20),(20, 10),(10, 30)' to equal '(10, 30),(40, 40),(40, 20),(20, 10)'.
      tests/wicket-gmap3-spec.js:834:35
```

It looks like maybe the first vertex is not being repeated (to close the ring)?

File: wicket-gmap3.js [hunk 1]
```diff
@@ -380,22 +371,6 @@
                             x: tmp.getAt(0).lng(),
                             y: tmp.getAt(0).lat()
                         });
-
```
Comment (songjiahong) [inline wicket-gmap3.js]: Here I'm not quite sure about the change, need your input here.
Does the components internally require all rings have the same orientation (clockwise or counter-clockwise)? If not, then maybe we don't need to reverse the inner rings

Comment (arthur-e) [reply wicket-gmap3.js]: Wicket doesn't care about the orientation of rings but the WKT standard requires that inner rings are reversed with respect to outer rings. For example, if the outer ring is oriented clockwise, then any inner rings should be oriented counter-clockwise.

**Comment by arthur-e** [comment]
Thanks for checking the tests @songjiahong. Let me know what you've decided about inner ring orientation, then I'll confirm the pull request.

File: wicket-gmap3.js (hunk 1 above)
Comment (songjiahong) [reply wicket-gmap3.js]: Think this through and update the multiple polygon check

**Comment by songjiahong** [comment]
All good now

**Comment by arthur-e** [comment]
Great! Thanks so much.

**Pull request merged**